### PR TITLE
fix: Grammatical errors

### PR DESCRIPTION
**Pull request by ArjunaKumarMohanta**
@JavkhlanK

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,8 +56,8 @@ be thoroughly reviewed.
 
 ### What changes we'll (most likely) not accept
 
-Most changes might be UI-related, and we think it's more or less a waste of time. If something design-related gets changed,
-ideally the whole app must follow the new style too, and that's hard to accomplish, especially for mods. That's why:
+Most changes might be UI-related, and we think it is more or less waste of time. If a something design-related gets changed,
```
Comment (khaled-0) [inline README.md:59]: ``If a``
Are you sure about your grammar bruh.

File: README.md [hunk 2]
```diff
@@ -94,11 +94,11 @@ Wanna chat with us, talk about changes, or just hang out? We have a Discord serv
 [![Join our Discord server!](https://invidget.switchblade.xyz/kq39yhT4rX)](http://discord.gg/kq39yhT4rX)
 
 ## Disclaimer
-This mod was not meant for any harmful purposes, such as harming Sketchware; Quite the opposite actually.
+This mod was not meant for any harmful purposes, such as harming Sketchware; Quite the opposite, actually.
 It was made to keep Sketchware alive by the community for the community. Please use it at your own discretion
-and be a Patreon backer of them, for example. Sadly, all other ways to support them aren't working anymore,
-so it's the only way available currently.
+and be a Patreon backer of them, for an example. Sadly, all other ways to support them are not working anymore,
```
Comment (khaled-0) [inline README.md:99]: No the an does not belong here

**Comment by khaled-0** [comment]
Lmao this commit just converts shorthand words to full form and introduces more grammatical mistakes.
Also the full stops aren't necessary.

Don't mind but know your stuff properly before trying to catch error of others

File: README.md (hunk 1 above)
Comment (PranavPurwar) [reply README.md:59]: also it should be: `more or less a waste of time`

**Comment by JavkhlanK** [comment]
Adding to that, please also don't mention any maintainers/contributors in PRs, at least not immediately after creating them. They are annoying.

**Pull request closed without merging**